### PR TITLE
Support CL pools in TX Fee module

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #https://github.com/osmosis-labs/osmosis/issues/6783

## What is the purpose of the change

Previously CL pools were not supported in tx fee module, as we were calculated sp via gamm keeper. 
This PR changes it to use pool manager keeper instead for sp calculation

## Testing and Verifying

This change added tests and can be verified as follows:

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

**Review by PaddyMc — APPROVED** [review]
Awesome, gg

File: x/txfees/keeper/feetokens_test.go [hunk 1]
```diff
@@ -65,7 +121,7 @@ func (s *KeeperTestSuite) TestUpgradeFeeTokenProposals() {
 		{
 			name:       "proposal with non-existent pool",
 			feeToken:   "foo",
-			poolId:     100000000000,
+			poolId:     10000000,
```
Comment (czarcas7ic) [inline x/txfees/keeper/feetokens_test.go:124]: Why was this changed?

Comment (mattverse) [reply x/txfees/keeper/feetokens_test.go:124]: in pool manager keeper we only use 12 bytes for storing pool id, this requires 13 bytes & causes panic in tests thus needs to be lowered!

**Review by ValarDragon — APPROVED** [review]

**Review by ValarDragon — APPROVED** [review]
